src/lua/uri.lua

```diff
@@ -133,6 +147,7 @@ local function format(uri, write_password)
     uribuf.path = uri.path
     uribuf.query = uri.query
     uribuf.fragment = uri.fragment
+    fill_uribuf_params(uribuf, uri)
```

Comment:
This also belongs to the commit that added query parameters to `struct uri`. Please rebase.

Reply:
I don't agree with you here, because without "params" table we can't test this part! Without "params" table, params after uri format is always equal to query!

Reply:
After discussion, we decided to leave it as it is